src/core/configuration-component.ts

```diff
@@ -19,15 +19,23 @@ interface IConfigurable extends IConfigurationOwner {
     $_config: Configuration;
 }
 
+function $_initBinding(vueInstance) {
```

Comment:
I would suggest to remove `$_` prefix from the function name. It is required only for custom private properties